CHANGELOG.md

```diff
@@ -110,8 +110,75 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - item images to the lists in the actor sheet
 - headings for the the lists in the inventory
 - weapons
+  - Alien Blaster
+  - Arcane Glove
+  - Auto-Axe
+  - Balefire Egg Launcher
+  - Baseball Bat
+  - Bladed Gauntlets
+  - Buckington Automatic Rifle M18
+  - Buckscatter G-M7 Launcher
+  - Dynamite
+  - Enclave Gale Blaster
+  - Enclave Hailgun
+  - Enclave Novasurge Rifle
+  - Enclave Rainmaker
+  - Enclave Thunderbuck
+  - Frag Grenade
+  - GAG Incinerator
+  - GAG Thunder Cannon
+  - Gauss Minigun
+  - Gauss Pistol
+  - Gauss Revolver
+  - Gauss Rifle
+  - Gauss Shotgun
+  - Gauss SMG
+  - Gretta M28 MG "Pecker"
+  - Gretta M33 MG "Predator"
+  - Gretta M3 SMG "Greaser"
+  - H&K MGat-39 "Mini Gun"
+  - Hellhound Gauntlets
+  - Hoof of Goddess
+  - Hunting Bow
+  - IF-44 "Angel Bunny"
+  - IF-84 "Stampede"
+  - IF-92 "Melrose"
+  - IF-G12 "Champion"
+  - Improvised Melee (small)
+  - Longbow
+  - Lumber Axe
+  - Mace
   - Machete
+  - Manticore Gauntlets
+  - Molotrot Cocktail
+  - No Weapon
+  - Plasma Grenade
+  - Power Glove
+  - Q-Same AEF-451 Flamer “Book Burner”
+  - Q-Same AEW-5 Arcane Pistol
+  - Q-Same AEW-6 Arcane Revolver
+  - Q-Same AEW-21 Arcane Rifle
+  - Q-Same AEW-33 Tri-Arcane Rifle
+  - Q-Same Arcane Gatling
+  - Q-Same MLEM
+  - Ripper
+  - SFO Launcher "Thumper"
+  - SFO Rocket Launcher "Boomer"
+  - Shishkebab
+  - Silaha Plasma "Corona"
+  - Silaha Plasma "Death Star"
+  - Silaha Plasma "Fusion"
+  - Silaha Plasma "Hestia"
+  - Silaha Plasma "Igneous"
+  - Silaha Plasma "Vulcan"
+  - Sledgehammer
+  - Sparkle-Nade
+  - Spear
   - Staff
+  - Super Sledge
+  - Thermic Lance
+  - Warhammer
+  - Yao Guai Gauntlets
```

Comment:
This needs to go under the heading of the next release ("Unreleased") version. And only the ones you added new.